.circleci/config.yml

```diff
@@ -186,6 +186,17 @@ defaults:
       - store_test_results: *store_test_results
       - store_artifacts: *artifacts_test_results
 
+  - test_ubsan_clang: &test_ubsan_clang
+      <<: *test_ubuntu2004_clang
+      steps:
+      - checkout
+      - attach_workspace:
+          at: build
+      - run:
+          <<: *run_soltest
+      - store_test_results: *store_test_results
+      - store_artifacts: *artifacts_test_results
+
```

Comment:
I have done the `test_san_clang` refactoring but not `test_asan` because the UBSan run is only executed on the clang build image.

Reply:
I really meant just renaming `test_asan` to match the naming here. This way the next person that needs to change this will see that it can be used with sanitizers in general and not just the address sanitizer.